.htaccess

```diff
@@ -70,7 +70,7 @@ FileETag MTime Size
 
 # gzip on Apache 2
 <IfModule mod_deflate.c>
-	AddOutputFilterByType DEFLATE text/html text/plain text/xml application/xml text/javascript text/css application/x-javascript application/xhtml+xml application/javascript
+	AddOutputFilterByType DEFLATE text/html text/plain text/xml application/xml text/javascript text/css application/x-javascript application/xhtml+xml application/javascript image/svg+xml
```

Comment:
@mathiashelin copy:fonts wordt gebruikt om het icon font te kopieren naar de Fonts-folder in Core en om al gegenereerde webfonts ook over te zetten.